chaosaws/ssm/actions.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+import random
+import re
+from typing import Any, Dict, List, Union
+
+import boto3
+from botocore.exceptions import ClientError
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import Configuration, Secrets
+from logzero import logger
+
+from chaosaws import aws_client
+from chaosaws.types import AWSResponse
+
+__all__ = ["create_document", "send_command", "delete_document"]
+
+
+def create_document(path_content: str = None,
+                    name: str = None,
+                    version_name: str = None,
+                    document_type: str = None,
+                    document_format: str = None,
+                    configuration: Configuration = None,
+                    secrets: Secrets = None) -> AWSResponse:
+    """
+    creates a Systems Manager (SSM) document.
+    An SSM document defines the actions that SSM performs on your managed.
+    For more information about SSM documents:
+    https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-ssm-docs.html
+    https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ssm.html#SSM.Client.create_document
+    """
+
+    if not any([path_content, name]):
```

Comment:
If you remove the default value of `path_content` and `name` in the signature, then this check will not be needed because missing values will be caught by the validation step of CTK. Though, it won't catch if someone sets these values to empty strings or None specifically.